apps/frontend/src/app/3_organisms/GettingStartedPopup/GettingStartedPopup.tsx

```diff
@@ -60,24 +60,24 @@ export const GettingStartedPopup: FC<GettingStartedPopupProps> = ({
         ],
         icon: IconNames.EARN_3,
       },
-      {
-        name: t(translationBasePath.lend.title),
-        description: t(translationBasePath.lend.description),
-        href: sovrynWikiLinks.lend,
-        icon: IconNames.LEND,
-      },
+      // {
```

Comment:
Delete the comments.

Reply:
We'll be adding these back in at some point soon (see story), but I think it's fine to remove since we'll have code in git history anyway

Reply:
I saw it in the description ofc but I've heard it so many times that I'd rather delete it and add it later if it's true :D